src/Zicht/Util/Html.php

```diff
@@ -185,7 +185,7 @@ private static function sanitizeTag($tag, $allowed = null, $resetStack = false)
                         $callback = $allowed;
                     } elseif (is_array($allowed) && isset($allowed[$tagName]) && is_callable($allowed[$tagName])) {
                         $callback = $allowed[$tagName];
-                    } elseif (is_array($allowed[$tagName]) && isset($allowed[$tagName][$attrName])) {
+                    } elseif (is_array($allowed) && is_array($allowed[$tagName]) && isset($allowed[$tagName][$attrName])) {
```

Comment:
Dit hoeft allemaal niet, onderstaande kan ook gewoon (in ieder geval in PHP 7.4):

```suggestion
                    } elseif (isset($allowed[$tagName][$attrName])) {
```

```
php > $x = 1; var_dump(isset($x['a']['b']));
bool(false)
```